### PR TITLE
Fix getting node id identifier - importer fixes

### PR DESCRIPTION
**Pull request by zerox1212**
Please review! It took me a long time to track this down and I don't know if this solution is correct.

The original code would make a second NodeId object instead of just getting the Identifier, but perhaps that is needed in some other case???

### CODE REVIEW

**Comment by zerox1212** [comment]
Example of what happens in this issue: https://github.com/FreeOpcUa/python-opcua/issues/317

File: opcua/common/xmlimporter.py [hunk 1]
```diff
@@ -15,18 +15,22 @@ def ua_type_to_python(val, uatype):
...
-        raise Exception("uatype nopt handled", uatype, " for val ", val)
+        raise Exception("ua type not handled", uatype, " for val ", val)
 
 
 def to_python(val, obj, attname):
     if isinstance(obj, ua.NodeId) and attname == "Identifier":
-        return ua.NodeId.from_string(val)
+        k, v = val.split("=", 1)
+        identifier = int(v)
+        return identifier
```
Comment (oroulet) [inline opcua/common/xmlimporter.py]: this is unfortunaltely wrong. For some strange reason what is written as identifier is the string version of a node....

**Comment by zerox1212** [comment]
What fix do you propose?

The problem is that when the extension object is instantiated (via this call `ext = getattr(ua, obj.objname)()`) the init creates a blank NodeId object. Then later `ua.NodeId.from_string(val)` gets called which returns a NumrericNodeId, but `setattr` seems to be assigning the __str__ value instead of the NumericNodeId object. This code is a little confusing for me to follow because there is a lot of builtin overrides, etc. Maybe the problem is the original NodeId from init is frozen and can't be changed?

**Comment by oroulet** [comment]
pushed a fix there: https://github.com/FreeOpcUa/python-opcua/pull/323

**Comment by oroulet** [comment]
btw I completely agree that the xml exporter and importer code is almost impossible to understand and hard to debug. I do not know why. Maybe it the nature of XML...

**Comment by zerox1212** [comment]
I tested your fix for the node id and it's working. Thanks.

Your fix does not include support for import extension object descriptions. Will you add that in your own PR or use this one?

**Comment by zerox1212** [comment]
I'm going to close this as it is no longer needed. However please add support for importing extension object descriptions (localized text) as soon as you get time. Thanks.

**Comment by oroulet** [comment]
Description works in the tests in tests_xml.py, look at code. is it broken
for you?

**Comment by oroulet** [comment]
@zerox1212 I added one more test to specifically check for description attribute of args in methods and it works. Do you still have issues? Maybe we still have a bug in xml format of description attribute?

**Comment by zerox1212** [comment]
I will try and open issue if needed. I'm on different project for the moment.

**Pull request closed without merging**